beacon-chain/rpc/beacon/validators.go

```diff
@@ -187,18 +188,24 @@ func (bs *Server) ListValidators(
 		requestedEpoch = q.Epoch
 	}
 
-	vals := headState.Validators
+	validatorList := make([]*ethpb.Validators_ValidatorContainer, 0)
```

Comment:
Implementing: https://github.com/prysmaticlabs/ethereumapis/pull/75 (including validator indices in listvalidators response)